lom/int_test/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Run 'make all' in the current directory
+make all
+
+# Check if the build succeeded
+if [ $? -eq 0 ]; then
+    echo "Build succeeded."
+
+    # Remove existing tar file if it exists
+    if [ -f "int_test.tar.gz" ]; then
+        rm int_test.tar.gz
+    fi
+
+    # Remove existing binaries in 'int_test/bin' if they exist
+    if [ -d "int_test/bin" ]; then
+        rm -rf int_test/bin/*
+    else
+        mkdir -p int_test/bin
+    fi
+
+    # Copy new files from 'build/bin' to 'int_test/bin'
+    cp -R build/bin/* int_test/bin/
```

Comment:
As these binaries are installed in container via debian package, no need for explicit copy
Your tar archive only carry test related files
At run time, we extract from archive into tmp folder inside container & run it.
At this point, engine & pluginmgr binaries are available in /usr/bin/

Reply:
As agreed , will revisit integration tests later